README.md

```diff
@@ -270,6 +271,18 @@ firstUser.watch(newState => console.log(`first user name: ${newState.name}`)) //
 addUser({name: Joseph}) // `firstUser` is not updated
 getUsers() // after promise resolve `firstUser` is updated and call all watchers (subscribers)
 ```
+Compose stores:
```

Comment:
Usually, we add link to repl to see examples in action in such way:

```markdown
[try it](https://share.effector.dev/2rgUVlLj)
```

[try it](https://share.effector.dev/2rgUVlLj)